docs/getting_started.rst

```diff
@@ -75,3 +75,45 @@ Subscribe to tag changes
 .. literalinclude:: ../examples/tag/subscribe_to_tag_changes.py
    :language: python
    :linenos:
+
+Data Frame API
+-------
+
+Overview
+~~~~~~~~
+
+The :class:`.DataFrameClient` class is the primary entry point of the Data Frame API.
+
+When constructing a :class:`.DataFrameClient`, you can pass an
+:class:`.HttpConfiguration` (like one retrieved from the
+:class:`.HttpConfigurationManager`), or let :class:`.DataFrameClient` use the
+default connection. The default connection depends on your environment.
+
+With a :class:`.DataFrameClient` object, you can:
+
+* Retrieve information about available API operations
+
```

Comment:
I'd be fine with leaving this one out. It's not going to be a common thing to do.

```suggestion
```